js/admin/src/utils/ExtensionRepository.js

```diff
@@ -60,4 +58,23 @@ export default class ExtensionRepository {
             m.endComputation();
         });
     }
+    toggleExtension(extension) {
+        const enabled = extension.enabled();
+
+        app.request({
+            url: app.forum.attribute('apiUrl') + '/extensions/' + id,
```

Comment:
I have to test that locally, but where is the id coming from ?

Plus, if we call the official `/extensions` endpoint, we need to use the shortName version of the package, not the packageId of Bazaar